client/src/main/java/io/pravega/schemaregistry/client/SchemaRegistryClientImpl.java

```diff
@@ -102,7 +112,7 @@ public boolean addGroup(String groupId, GroupProperties groupProperties) {
     @Override
     public void removeGroup(String groupId) {
         withRetry(() -> {
-            Response response = groupProxy.deleteGroup(groupId);
+            Response response = groupProxy.deleteGroup(groupId, namespace);
```

Comment:
`namespace`, then `groupId` looks like a more natural method params order. Even more so with proxies. Can we change it in `deleteGroup()` and in other places?